.github/workflows/screener-run.yml

```diff
@@ -1,32 +1,49 @@
 name: Screener run
-
 on:
-  workflow_run:
-    workflows:
-      - Screener build
-    types:
-      - completed
+  pull_request:
 
+  #workflow_run:
+  #  workflows:
+  #    - Screener build
+  #  types:
+  #    - completed
 env:
   AZURE_STORAGE_CONNECTION_STRING: ${{secrets.AZURE_STORAGE_CONNECTION_STRING}}
-
 jobs:
   screener-react-northstar:
     runs-on: 'ubuntu-latest'
-
     name: Screener @fluentui/react-northstar
-
     steps:
       - uses: actions/checkout@v3
         with:
-          ref: ${{github.event.workflow_run.head_branch}}
           fetch-depth: 0
 
       - uses: actions/setup-node@v3
         with:
           node-version: 14.18.1
           cache: 'yarn'
 
+      - name: Check if test app artifact deployed
+        uses: actions/github-script@v6
+        id: skip-screener
+        with:
+          script: |
+            let allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               run_id: 2917202539,
```

Comment:
used a previous run again for testing purposes, will be replaced by the run id of the triggering workflow